### ISSUE TITLE
Lint/AmbiguousRegexpLiteral: auto-correction produces invalid code

### ISSUE THREAD

**Issue by deepj** [rubocop-hq/rubocop#8843]
It seems it fails on the regexp literal with method invocation.

## Expected behavior

Auto-corrected from `assert /#{device_sn_pattern}/.match('1234')` to `assert(/#{device_sn_pattern}/.match('1234'))`

## Actual behavior

```ruby
assert /#{device_sn_pattern}/.match('1234'))
```

Left parenthesis missing after `assert`. It breaks Ruby syntax

## Steps to reproduce the problem

Code snippet

```ruby
assert /#{device_sn_pattern}/.match('1234')
```

Run

```
rubocop --only Lint/AmbiguousRegexpLiteral -a
```

## RuboCop version

```
$ rubocop -V
0.92.0 (using Parser 2.7.1.5, rubocop-ast 0.7.1, running on ruby 2.7.1 x86_64-darwin18)
```
